samples/csharp_dotnetcore/49.qnamaker-all-features/Dialog/QnAMakerBaseDialog.cs

```diff
@@ -63,9 +67,49 @@ protected async override Task<QnADialogResponseOptions> GetQnAResponseOptionsAsy
                 CardNoMatchText = DefaultCardNoMatchText,
                 NoAnswer = noAnswer,
                 CardNoMatchResponse = cardNoMatchResponse,
+                DisplayPreciseAnswerOnly = this.DisplayPreciseAnswerOnly
             };
 
             return responseOptions;
         }
+        private bool EnablePreciseAnser
```

Comment:
spelling

Reply:
Modified.

---
In reply to: [430667510](https://github.com/microsoft/BotBuilder-Samples/pull/2430#discussion_r430667510) [](ancestors = 430667510)